tldw_Server_API/tests/Config/test_config_providers_endpoints.py

```
@@ -0,0 +1,360 @@
+        ):
+            response = await list_configured_providers()
+
+        openai_item = next(p for p in response.providers if p.name == "openai")
+        assert openai_item.configured is True
+        assert openai_item.key_hint is not None
+        assert "sk-" in openai_item.key_hint
+        assert openai_item.key_source == "env"
+        assert response.any_configured is True
+
+    @pytest.mark.asyncio
+    async def test_local_providers_always_configured(self, monkeypatch):
```

Ruff (0.15.7)

[warning] Unused method argument: `monkeypatch`

(ARG002)